website/edit.php

```diff
@@ -218,14 +218,14 @@ function edit_put($sql) {
 
     // haslo starego typu
     if ($haslo != '') {
```

Comment:
haslo haslo2 told nothing to me :  maybe could we move line 213->224 in a dedicated method that return `oldPasswordVerificationResult`

Reply:
Seems to be an overhead for me but feel free to provide PR on new theme branch :D

Reply:
Thinking at it again, the `longin.php` should have some equivalent code, so yes it would need refactor :+1:

Reply:
Btw new theme was started just to change the theme, but more changes are included. It s like a general first pass refactor now :(

Reply:
so keep untouched no worry ;)

Reply:
(I had not seend your latest commit :+1:)